tests/run-make/reproducible-build-2/reproducible-build.rs

```diff
@@ -94,13 +89,15 @@ fn main() {
         f(0);
     }
 
-    with_fn_once_adapter(|_:i32| { });
+    with_fn_once_adapter(|_: i32| {});
 
     reproducible_build_aux::regular_fn(STATIC);
     reproducible_build_aux::generic_fn::<u32, char>();
     reproducible_build_aux::generic_fn::<char, Struct<u32, u64>>();
-    reproducible_build_aux::generic_fn::<Struct<u64, u32>,
-                                         reproducible_build_aux::Struct<u32, u64>>();
+    #[rustfmt::skip] // rustfmt formats this as longer than 100 chars, which tidy doesn't like!
```

Comment:
The `// ignore-tidy-linelength` directive may work to avoid doing this

Reply:
Ok, I have switched to that.